bin/lint

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# Check our code base for inconsistencies in code formatting
+# and common mistakes.
+#
+# Run this command with:
+#   ./bin/lint
+
+PATH=$(yarn bin):$PATH
+
+# Check these files for mistakes:
+SRC=$(git ls-files | grep -v "^\." | grep ".js\$")
+
+# Prettier is an automated code formatter.
+# https://github.com/prettier/prettier
+prettier --semi false --single-quote --list-different $SRC
```

Comment:
ah, i think it should not halt the script personally, so you get all the warnings at once instead of 1. fixing prettier things, then 2. fixing linter warnings